typings/index.d.ts

```diff
@@ -3911,7 +3917,7 @@ export interface IntegrationAccount {
   name: string;
 }
 
-export interface InteractionCollectorOptions<T extends Interaction> extends CollectorOptions<[T]> {
+export interface InteractionCollectorOptions<T extends Interaction> extends CollectorOptions<T[]> {
```

Comment:
This will make it loose strictness and accuracy... It should stay as a tuple, or otherwise always pass in a tuple to T

Reply:
Good point, that has been fixed.